.actor/input_schema.json

```diff
@@ -136,11 +136,14 @@
             "description": "If enabled, the Actor attempts to close or remove cookie consent dialogs to improve the quality of extracted text. Note that this setting increases the latency.",
             "default": true
         },
-        "useCheerioCrawler": {
-            "title": "Use Cheerio Crawler",
-            "type": "boolean",
-            "description": "If enabled, the Actor uses the Cheerio Crawler to extract the target web page content.",
-            "default": false
+        "scrapingTool": {
+            "title": "Which scraping tool to use",
+            "type": "string",
+            "description": "Choose what scraping tool to use for extracting the target web pages. The Browser tool is more powerful and can handle JavaScript heavy websites. While the Plain HTML tool is about two times faster.",
+            "editor": "select",
+            "default": "playwright",
+            "enum": ["playwright", "cheerio"],
```

Comment:
I wanted to propose to sync these to `enumTitles` because if these are different, it creates mess with debugging and discussion later. But maybe it would be better to expand the `enumTitles` as `Browser (uses Playwright)`?